src/main/java/fact/auxservice/AuxPoint.java

```diff
@@ -12,11 +13,11 @@
  * Created by kai on 31.03.15.
  */
 public class AuxPoint implements Comparable<AuxPoint>{
-    private DateTime timeStamp;
+    private OffsetDateTime timeStamp;
     private ImmutableMap<String, Serializable> data;
 
-    public AuxPoint(DateTime timeStamp){
-        this.timeStamp = new DateTime(timeStamp);
+    public AuxPoint(OffsetDateTime timeStamp){
+        this.timeStamp =OffsetDateTime.of(timeStamp.toLocalDate(),timeStamp.toLocalTime(), ZoneOffset.of("+00:00"));
```

Comment:
Why is it neccessary to create a new object here? It's basically a copy isn't it?